packages/@okta/vuepress-site/docs/guides/oie-upgrade-api-sdk-to-oie-sdk/main/index.md

```diff
@@ -660,10 +660,10 @@ curl --location --request POST 'https://${yourOktaDomain}/api/v1/authn/credentia
 }
 ```
 
-If your application code implements these API calls and handles the responses shown, you need to update your code to use Identity Engine SDK methods. These methods encapsulate the password recovery flow using recursive calls to the Identity Engine, and a successful response returns with access and ID tokens.
+If your application code implements these API calls and handles the responses shown, you need to update your code to use Identity Engine SDK methods. These methods encapsulate the password recovery flow using recursive calls to Identity Engine, and a successful response returns with access and ID tokens.
 
-See [Okta Identity Engine SDK authentication flow for password recovery](#okta-identity-engine-sdk-authentication-flow-for-password-recovery).
+See [Identity Engine SDK authentication flow for password recovery](#identity-engine-sdk-authentication-flow-for-password-recovery).
```

Comment:
This note doesn't work for asp.net and ios because there is no "Identity Engine SDK authentication flow for password recovery" section for those two languages in the `pswrvy.md` file. I also checked in the "live" docs just for giggles and the notes don't work there either for those languages. This notes should be moved into the `pswrvy.md` file for the languages that include this section, rather than in the main index file.